src-java/kilda-persistence-tinkerpop/src/test/java/org/openkilda/persistence/ferma/repositories/FermaFlowRepositoryTest.java

```diff
@@ -91,8 +94,49 @@ public void setUp() {
 
         switchA = createTestSwitch(TEST_SWITCH_A_ID.getId());
         switchB = createTestSwitch(TEST_SWITCH_B_ID.getId());
+        switchC = createTestSwitch(TEST_SWITCH_C_ID.getId());
 
-        assertEquals(2, switchRepository.findAll().size());
+        assertEquals(3, switchRepository.findAll().size());
+    }
+
+    @Test
+    public void findPortToFlowsMap() {
```

Comment:
I'd change the name to something like "shouldFindSwitchFlowsByPortWithoutFilter" to keep the naming convection in this class

Reply:
Actually, I wanted to do the opposite: remove _should_ from all test cases 😃 
Should in English means that it is up to a person to do it or not. Like as in those user stories types: must and should. It's ok to deliver a feature without "should" user stories, just nice to have, but not mandatory.
To me, _should find_ sounds like "it's ok if nothing is found" or some sort of hesitation: _maybe you'll find, I don't know, fingers crossed._

I'd vote for either not use _should_ at all (in the end it doesn't bring any new information), or add _must_ instead (this method must do what it supposed to do, otherwise it is a defect that needs to be fix). 
Another option might be given-when-then pattern, or a simplified version when-then. But we have strict rules about line length, so good descriptive names are no-go due to checkstyle 😒